Configuration.md

```diff
@@ -1,149 +1,167 @@
 # Java Debug Configuration
 
 ## Table of Contents
-* [Auto generate the launch.json](#auto-generate-the-launchjson)
-* [Modify the launch.json](#modify-the-launchjson)
-  * [Main](#main)
-    * mainClass
-    * projectName
-  * [Arguments](#arguments)
-    * args
-    * vmArgs
-  * [Environment](#environment)
-    * console
-    * env
-  * [Don't step into the specified classes or methods](#dont-step-into-the-specified-classes-or-methods)
-    * stepFilters
-  * [Attach to a debuggee](#attach-to-a-debuggee)
-    * hostName
-    * port
-    * [Attach to mvn task](#attach-to-mvn-task)
-    * [Attach to embedded maven tomcat server](#attach-to-embedded-maven-tomcat-server)
-    * [Use javac as the builder and attach to java process](#use-javac-as-the-builder-and-attach-to-java-process)
-* [Modify the settings.json (User Setting)](#modify-the-settingsjson-user-setting)
-  * java.debug.settings.console
-  * java.debug.settings.forceBuildBeforeLaunch
-  * java.debug.settings.hotCodeReplace
-  * java.debug.settings.enableRunDebugCodeLens
-* [FAQ](#faq)
-
-
-The debugger provides two kinds of configuration: *launch.json* and *settings.json* (User Settings), see the [README](https://github.com/Microsoft/vscode-java-debug#options) for the supported configuration list. launch.json is used to control the configuration per debug session, and the user setting is shared by the whole workspace or VS Code.
+

```

Comment:
Did you change anything in this file?

Reply:
I did - added the documentation for the new config setting. I guess auto formatting messed the compare.

Reply:
e.g. 

line 27:
```md
    -   java.debug.settings.onBuildFailure
```

and...

line 374:
```md
-   `java.debug.settings.onBuildFailure` - Sometimes you may be bothered with the message _"Build failed, do you want to continue?"_, you could use this setting to suppress the message with a specific selection.
```

Reply:
The PR should keep the minimum changes. It's better to keep the original formatter, and just add your changes.

Reply:
done